src/objectExplorer/objectExplorerService.ts

```diff
@@ -412,9 +412,19 @@ export class ObjectExplorerService {
         const newConnectionGroupNodes = new Map<string, ConnectionGroupNode>();
         const newConnectionNodes = new Map<string, ConnectionNode>();
 
+        // Read user setting for default expansion state
+        const config = this._vscodeWrapper.getConfiguration(Constants.extensionConfigSectionName);
+        const expandGroupsByDefault = config.get<boolean>(
+            "connectionGroupsExpandedByDefault",
```

Comment:
Can you move this constants.ts file?
https://github.com/microsoft/vscode-mssql/blob/main/src/constants/constants.ts.
For example: 
https://github.com/microsoft/vscode-mssql/blob/5cc8621043de3e876d09f52a950f3e4c8c766f77/src/constants/constants.ts#L237